package.json

```diff
@@ -64,17 +64,17 @@
     "@adonisjs/core": "^5.1.0"
```

Comment:
I think we have to upgrade the peer dependency to @adonisjs/core to at least 5.5.1